content/en/docs/concepts/storage/volumes.md

```diff
@@ -367,69 +326,71 @@ spec:
 
 ### fc (fibre channel) {#fc}
 
-An `fc` volume allows an existing fibre channel volume to be mounted in a Pod.
-You can specify single or multiple target World Wide Names using the parameter
+An `fc` volume type allows an existing fibre channel block storage volume to be mounted in a Pod.
+You can specify single or multiple target world wide names (WWNs) using the parameter
 `targetWWNs` in your volume configuration. If multiple WWNs are specified,
 targetWWNs expect that those WWNs are from multi-path connections.
 
-{{< caution >}}
+{{< note >}}
 You must configure FC SAN Zoning to allocate and mask those LUNs (volumes) to the target WWNs beforehand so that Kubernetes hosts can access them.
-{{< /caution >}}
+{{< /note >}}
 
-See the [FC example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/fibre_channel) for more details.
+See the [fibre channel example](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/volumes/fibre_channel) for more details.
 
-### flocker {#flocker}
+### flocker
```

Comment:
we should probably mark this as deprecated/unsupported

Reply:
OK. I will update. When should a deprecated volume type be removed from the documentation?